deploy-monitoring-services.md

```diff
@@ -28,19 +28,19 @@ summary: 本文适用于手动部署 TiDB 监控报警系统的用户。假设 T
 {{< copyable "shell-regular" >}}
 
 ```bash
-wget https://download.pingcap.org/prometheus-2.27.1.linux-amd64.tar.gz
+wget https://download.pingcap.org/prometheus-2.49.1.linux-amd64.tar.gz
```

Comment:
this link does not work

Reply:
Official Download Link： "https://github.com/prometheus/prometheus/releases/download/v${prometheus_ver}/prometheus-${prometheus_ver}.${prometheus_os}-${prometheus_arch}.tar.gz"